jac-uk/apply
# #1053 Bugfix: Sign in from the vacancy details advert automatically starts an application

## What's included?
Closes #1053 

## Who should test?
✅ Product owner
✅ Developers

## How to test?
See bug ticket for the steps to replicate the original problem HOWEVER please also try to break sign in, sign up, apply, try to access protected pages (e.g. /applications) whilst logged out, page refresh etc. Anything you can think of to try to break the navigation.

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
